bmaptools/BmapHelpers.py

```diff
@@ -83,3 +94,72 @@ def program_is_available(name):
             return True
 
     return False
+
+
+def get_mount_point(filepath):
+    """
+    Return the mount point the file object file resides on.
+    """
+    f = os.path.realpath(filepath)
+    while f != os.path.sep:
+        if os.path.ismount(f): return f
+        f = os.path.realpath(os.path.join(f, os.pardir))
+    return f
+
+
+def get_file_system_type(filepath):
```

Comment:
FYI, in python we use leading underscore to communicate that a global symbol is private to this module and should not be used outside of the module. I do not know if this entire project is careful about it, but at least for this and other functions you add, mark them as private by using this name: _get_file_system_type().Those functions that you do want to be visible outside of this module (BmapHelpers) should start with a non-_.